src/test/java/poll/service/VoteServiceTest.java

```diff
@@ -1,4 +1,78 @@
 package poll.service;
 
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import poll.repository.Answer;
+import poll.repository.Vote;
+import poll.repository.VoteRepository;
+
+import java.time.LocalDateTime;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.when;
+
+
+@ExtendWith(MockitoExtension.class)
 public class VoteServiceTest {
+
+    @Mock
+    VoteRepository voteRepository;
+
+    @InjectMocks
+    VoteService voteService;
+
+
+    @Test
+    public void should_ReturnVote_when_GivenVoteId() {
+        //given
+        Vote expectedVote = new Vote(3l, "Ivan Zaitsev", LocalDateTime.of(2021, 1, 2, 9, 30), Answer.YES);
+        //when
+        when(voteRepository.findById(3l)).thenReturn(Optional.of(expectedVote));
```

Comment:
I can see you decided to use the given-when-then conventionn which is completely fine. But in this scenario the Mockito's when() method is part of test's environment preparation, so I think this should be moved to the "given" section. In "when" section there should only be the service call, so sth like:
`Optional<Vote> result = voteService.getVoteById(3l);`

and the result variable should be used in "then section"